tests/azure/steps-run-tests.yml

```diff
@@ -35,6 +37,16 @@ steps:
         ./build/simple_test_from_binary
         ./build/simple_test_from_binary_static
       displayName: Offline compilation simple tests
+    - script: |
+        set -ex
+        clang  -Xclang -finclude-default-header -c -target spir -O0 -emit-llvm -o simple-cl.bc ./tests/simple-from-il-binary/simple.cl
```

Comment:
I'm wondering whether this should build and use the clang coming from the LLVM tree that clspv uses to avoid spurious CI failures because of incompatibilities between whatever version of clang is available on the CI runners and the ToT version of llvm-spirv. Unclear to me how much of a practical problem this could be.